qa/pull-tester/rpc-tests.py

```diff
@@ -37,7 +38,7 @@
     # terminal via ANSI escape sequences:
     BOLD = ('\033[0m', '\033[1m')
 
-RPC_TESTS_DIR = BUILDDIR + '/qa/rpc-tests/'
+RPC_TESTS_DIR = SRCDIR + '/qa/rpc-tests/'
```

Comment:
Note to myself: Does the cache (from `create_cache.py` et al.) end up in SRCDIR now?